api-reference/_hidden/dxFileManagerToolbar/items/items.md

```diff
@@ -23,30 +42,93 @@ The **FileManager** widget allows you to add default and create custom toolbar i
         $("#file-manager").dxFileManager({
             toolbar: {
                 items: [
+                    // Specify a predefined item's name and optional settings
                     {
-                        name: "showNavPane",
-                        visible: true
+                        name: "create",
+                        text: "Create a directory",
+                        icon: "newfolder"
                     },
-                    "separator", "create",
+                    // Specify a predefined item's name only                    
+                    "switchView", "separator"
+                    //...
+                ]
+                fileSelectionItems: [
+                    "copy", "rename"
+                    //...
+                ]
+            }
+        });
+    });
+
+---
+
+**Custom Items**
+
+To add a custom toolbar item, specify its [text](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/items/#text) and optional settings (for example, a file extension for the toolbar item that creates a new file) to one of the following collections:
+
+- [items](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/items/) - Displays toolbar items when no one file system item is selected.
+
+- [fileSelectionItems](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/fileSelectionItems/) - Displays toolbar items when one or more file system items are selected.
+
+The [widget](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/items/#widget) option allows you to specify a widget for a custom toolbar item ([dxButton](/Documentation/ApiReference/UI_Widgets/dxButton/) is used by default). Use the [toolbarItemClick](/Documentation/ApiReference/UI_Widgets/dxFileManager/Events/#toolbarItemClick) event to handle clicks on custom toolbar items.
```

Comment:
Why do you add "by default"?

Reply:
If you do not specify the widget option, the dxButton widget is used as default widget.

Reply:
```suggestion
The [widget](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/toolbar/items/#widget) option allows you to specify a widget for a custom toolbar item ([dxButton](/Documentation/ApiReference/UI_Widgets/dxButton/) is the default widget). Use the [toolbarItemClick](/Documentation/ApiReference/UI_Widgets/dxFileManager/Events/#toolbarItemClick) event to handle clicks on custom toolbar items.
```